media/css/pebbles/components/_buttons-download.scss

```diff
@@ -66,6 +67,7 @@ ul.download-list {
 
 .download-button {
     display: inline-block;
+    text-align: center;
```

Comment:
This is going to change all text and messaging within the download button to centered, which looks odd on some pages where text is left or right aligned. Is there a reason to change it?

Reply:
It aims to align the download button and the Privacy Notice link. Without this, [it looks rather odd](https://screenshots.firefox.com/t286AJZDpqHVxol2/localhost).

Reply:
I'm wondering if this message really needs to be part of the download button itself, or if it would be enough just to include this as a message on our main Firefox download pages. See https://github.com/mozilla/bedrock/pull/6044#discussion_r213969504